packages/generator-honeycomb/app/templates/src/server/views/layouts/default.js

```diff
@@ -1,25 +0,0 @@
-/* eslint-disable react/no-danger */
```

Comment:
i had created the template because of 
```
<style type="text/css">		
    {styleSheet.rules().map(rule => rule.cssText).join('\n')}		
</style>
```

to keep this logic in one file - maybe we need it again, if we can use styled-components with preact